pkg/util/disk.go

```diff
@@ -0,0 +1,50 @@
+package util
+
+import (
+	"fmt"
+	"strconv"
+)
+
+const (
+	MinDiskSize       = 250 << 30
+	MinPersistentSize = 150 << 30
+	MiByteMultiplier  = 1 << 20
+	GiByteMultiplier  = 1 << 30
+
+	// 50Mi for COS_OEM, 15Gi for COS_STATE, 8Gi for COS_RECOVERY, 64Mi for ESP partition, 50Gi for VM data
+	fixedOccupiedSize = (50 + 15360 + 8192 + 64 + 51200) * MiByteMultiplier
+)
+
+func ParsePartitionSize(diskSizeBytes uint64, partitionSize string) (uint64, error) {
```

Comment:
prefer to hide the partition size validation when the data disk is not an installation disk, in the meanwhile, the installation disk size is valid.
![image](https://github.com/harvester/harvester-installer/assets/4569037/15caf8e4-ba53-4768-ae6a-8e824b66e218)